src/coreclr/gc/gc.cpp

```diff
@@ -33113,6 +33156,8 @@ void gc_heap::background_mark_phase ()
     bgc_t_join.join(this, gc_join_concurrent_overflow);
     if (bgc_t_join.joined())
     {
+// TODO: Can we simply get rid of the join - fine for running - but how about tooling?
```

Comment:
I don't understand this comment. what about tooling? 

but I don't think this join is needed for regions.